tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -345,7 +354,9 @@ test = Matrix(test_cut[:, remove_names]);
 
     # Calculate all the mu terms.
     mu = intercept .+ x * coefficients
-    return y ~ MvNormal(mu, σ² * I)
+    for i in 1:n_obs
+        y[i] ~ Normal(mu[i], sqrt(σ²))
+    end
```

Comment:
Is there a particular reason why this change was made? Why are we not using the vectorized implementation?

Reply:
I think it might be useful to make explicit use of `n_obs` as it is set as an argument in the old version. Also may be more readable and insutitive for new commers (as many Stan/PyMC3 tutorials do).
Admittedly, vectorised version is more efficient - I just recerted to that.